src/Sarif.Viewer.VisualStudio/ProjectNameLookup.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Sarif.Viewer
+{
+    public sealed class ProjectNameLookup
+    {
+        private readonly static ProjectNameLookup _instance = new ProjectNameLookup();
+        private Dictionary<string, string> _projectNames = new Dictionary<string, string>();
```

Comment:
>_projectNames [](start = 43, length = 13)

We don't use the leading underscore convention (actually I like that convention, but I can't have everything I want...)